test/surface_test.py

```diff
@@ -1586,12 +1586,12 @@ def test_src_alpha_sdl2_blitter(self):
                         src_surf, (0, 0), special_flags=pygame.BLEND_ALPHA_SDL2
                     )
                     key = ((dst_r, dst_b, dst_a), (src_r, src_b, src_a))
-                    results[key] = dest_surf.get_at((65, 33))
-                    self.assertEqual(results[key], results_expected[key])
+                    results[key] = tuple(dest_surf.get_at((65, 33)))
+                    for i in range(4):
+                        self.assertAlmostEqual(results[key][i], results_expected[key][i], None, None, 4)
```

Comment:
Instead of passing two `None`s like that (which would not work for the 3rd argument of this function which needs an integer), you could instead pass the `4` as a keyword argument like so
```suggestion
                        self.assertAlmostEqual(results[key][i], results_expected[key][i], delta=4)
```

Reply:
Resolved that as well now